lib/SIL/TypeLowering.cpp

```diff
@@ -2109,8 +2111,17 @@ TypeConverter::hasLoweredLocalCaptures(SILDeclRef fn) {
   return !getLoweredLocalCaptures(fn).getCaptures().empty();
 }
 
+static bool cannotCapture(DeclContext *DC) {
```

Comment:
Since this appears twice it should be a method on the TypeLowering object. Also it should take a SILDeclRef or AnyFunctionRef

Reply:
Yes, the duplication in the checks is the main thing I need to clean up (if this is the correct check).